src/runtime/pkg/govmm/qemu/qemu.go

```diff
@@ -15,7 +15,9 @@ package qemu
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
+	"github.com/kata-containers/kata-containers/src/runtime/pkg/device/api"
```

Comment:
I noticed that you're only using this for logging purposes.
Please, use `log`, which is already imported here, instead of importing a new package.